contracts/minting/Controller.sol

```diff
@@ -40,7 +40,7 @@ contract Controller is Ownable {
      * @dev ensure that the caller is the controller of a non-zero worker address
      */
     modifier onlyController() {
-        require(controllers[msg.sender] != address(0));
+        require(controllers[msg.sender] != address(0), "Sender must be a controller.");
```

Comment:
I think we should never convert to all upper-case. All addresses coming back from Web3 will be checksummed (mixed-case), so we need to make sure the addresses we have hardcoded are also checksummed, and then if we ever get an address from a source other than Web3, use `web3.utils.toChecksum` (https://web3js.readthedocs.io/en/1.0/web3-utils.html#tochecksumaddress) on it.

Reply:
> I think we should never convert to all upper-case. All addresses coming back from Web3 will be checksummed (mixed-case), so we need to make sure the addresses we have hardcoded are also checksummed, and then if we ever get an address from a source other than Web3, use `web3.utils.toChecksum` (https://web3js.readthedocs.io/en/1.0/web3-utils.html#tochecksumaddress) on it.

If things are coming back as checksummed and not uppercase, this would be a great approach.